Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalGammaJet_Output_cff.py

```diff
@@ -8,12 +8,37 @@
         SelectEvents = cms.vstring('pathALCARECOHcalCalGammaJet')
     ),
     outputCommands = cms.untracked.vstring( 
-                 'keep recoPhotonCores_*_*_*',
-                 'keep recoSuperClusters_*_*_*',
+                 #'keep recoPhotonCores_*_*_*',
+                 'keep recoPhotonCores_gedPhotonCore_*_*',
+                 'keep recoPhotonCores_photonCore_*_*',
+                 #'keep recoSuperClusters_*_*_*',
+                 'keep recoSuperClusters_SCselector_*_*',
+                 'keep recoSuperClusters_cleanedHybridSuperClusters_*_*',
+                 'keep recoSuperClusters_correctedHybridSuperClusters_*_*',
+                 'keep recoSuperClusters_correctedMulti5x5SuperClustersWithPreshower_*_*',
+                 'keep recoSuperClusters_hfEMClusters_*_*',
+                 'keep recoSuperClusters_hybridSuperClusters_*_*',
+                 'keep recoSuperClusters_multi5x5SuperClustersWithPreshower_*_*',
+                 'keep recoSuperClusters_particleFlowEGamma_*_*',
+                 'keep recoSuperClusters_uncleanedHybridSuperClusters_*_*',
+                 'keep recoSuperClusters_uncleanedOnlyCorrectedHybridSuperClusters_*_*',
+                 'keep recoSuperClusters_uncleanedOnlyCorrectedMulti5x5SuperClustersWithPreshower_*_*',
+                 'keep recoSuperClusters_uncleanedOnlyMulti5x5SuperClustersWithPreshower_*_*',
+                 'keep recoSuperClusters_multi5x5SuperClustersCleaned_*_*',
+                 'keep recoSuperClusters_multi5x5SuperClustersUncleaned_*_*',
+                 'keep recoSuperClusters_multi5x5SuperClusters_*_*',
+                 'keep recoSuperClusters_particleFlowSuperClusterECAL_*_*',
```

Comment:
This removed the following branches:
```
recoSuperClusters_reducedEgamma_reducedSuperClusters_*.
recoPhotonCores_reducedEgamma_reducedGedPhotonCores_*.
recoSuperClusters_mergedSuperClusters__*.
```
based on modified test derived from 134.911, adding HcalCalDijets+HcalCalGammaJet to step3.

@jeongeun 
Please make sure that your test config is running relevant ALCAs, if they are modified.